Sources/Common/Core/Math/index.js

```diff
@@ -1995,6 +2044,93 @@ export function solve3PointCircle(p1, p2, p3, center) {
   return radius;
 }
 
+export function computeHistogram(imageData, worldBounds, voxelFunc = null) {
```

Comment:
it is too much 'imageData" specific to be in the "generic" Math/index.js file.
Maybe move to `ImageData/index.js` ?